types/knockout.mapping/index.d.ts

```diff
@@ -12,11 +13,36 @@ export as namespace mapping;
 declare var self: KnockoutMapping;
 export = self;
 
+
 declare global {
-    type KnockoutObservableType<T> = {	
-        [P in keyof T]: KnockoutObservable<KnockoutObservableType<T[P]>>|T[P];	
+    // the idea behind this is good, but doesn't work as intended. This will make object properties T | Observable<T>, 
```

Comment:
Agree. It must be possible to use conditional types here. Do you want to try them?